File: LeetCode/Go/node.go
package leetcode

// ListNode defination for singly-linked list
type ListNode struct {
	Val  int
	Next *ListNode
}

// TreeNode tree node of bin tree
type TreeNode struct {
	Val   int
	Left  *TreeNode
	Right *TreeNode
}

// Node definition for a Node
type Node struct {
	Val      int
	Children []*Node
}

File: LeetCode/Python/node.py
from typing import Optional


class ListNode:
    def __init__(self, val: any = 0, next: Optional["ListNode"] = None) -> None:
        self.val = val
        self.next = next


class TreeNode:
    def __init__(
        self,
        val: any = 0,
        left: Optional["TreeNode"] = None,
        right: Optional["TreeNode"] = None,
    ) -> None:
        self.val = val
        self.left = left
        self.right = right
